share/functions/__fish_preview_current_file.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_preview_current_file --description "Open the current file in a pager"
+    # set pager to user-preferred pager or less
+    set -l pager "less --"
```

Comment:
Make this a more idiomatic array (`set -l pager less --`)
That'd be consistent with the next line, since that uses `read -a`.

Reply:
done